docs/Developer-Guide.md

```diff
@@ -212,11 +212,15 @@ $ sudo systemctl restart systemd-journald
 >
 > - You should only do this step if you are testing with the latest version of the agent.
 
-The rust-agent is built with a static linked `musl.` To configure this:
+The agent is built with a statically linked `musl.` The default libc used is `musl`, but on `ppc64le` and `s390x`,
+`gnu` should be used. To configure this:
 
 ```
-rustup target add x86_64-unknown-linux-musl
-sudo ln -s /usr/bin/g++ /bin/musl-g++
+$ export ARCH=$(uname -m)
+$ [ ${ARCH} == "ppc64le" ] || [ ${ARCH} == "s390x" ] && export LIBC=gnu || export LIBC=musl
```

Comment:
There are several problems with this line:

* If `$ARCH` is not defined, then you get an error, `bash: [: ==: unary operator expected`
* It does not parse for someone running `zsh`: `zsh: = not found`
* The braces around `${ARCH}` add nothing to readability IMHO
* The sequence of logical operators, some being used as tests, other being used as statements, is "cute" but really more obscure than we would want in a documentation.

For these reasons, I suggest the following alternative:

```
 if [ "$ARCH" = "ppc64le" -o "$ARCH" = "s390x" ]; then export LIBC=gnu; else export LIBC=musl; fi
```

This is assuming that we want to default to `musl` if `$ARCH` is not set.

Reply:
Thanks @c3d , it's better.